pkg/api/k8s_versions.go

```diff
@@ -66,9 +66,9 @@ const (
 	ScheduledMaintenanceManagerImageReference         string = "quay.io/awesomenix/drainsafe-manager:latest"
 )
 
-// k8sComponentVersions is a convenience map to make UT maintenance easier,
-// at the expense of some add'l indirection in getK8sVersionComponents below
-var k8sComponentVersions = map[string]map[string]string{
+// k8sComponentVersionsGCR is a convenience map to make UT maintenance easier,
+// at the expense of some add'l indirection in getK8sVersionComponentsGCR below
+var k8sComponentVersionsGCR = map[string]map[string]string{
```

Comment:
Here's the meat of the refactor. We rename our existing static dictionaries to reflect that they are gcr-specific.